index.js

```diff
@@ -50,6 +50,8 @@ async function run() {
         } else {
             console.log("Node js module is up to date.");
         }
+        await git.pull(branch, ['-f']);
```

Comment:
This doesn't seem ideal. Is there no way for us to do this "locally"? Keep in mind that while this action is running, someone may have pushed more commits onto the PR branch.

/cc @Purushothaman22

Reply:
This line adds no value as long as your running the CI locally for you to fetch the latest commit.  You have to do it locally.

Reply:
So a `git reset --hard` should be fine?

Reply:
> So a git reset --hard should be fine?

Yes.